### ISSUE TITLE
module error

### ISSUE THREAD

**Issue by azhang** [maticnetwork/eth-dagger.js#2]
```
var Dagger = require('eth-dagger')
new Dagger(...)
```
doesn't work for me, I have to use
```
var Dagger = require('eth-dagger').default
new Dagger(...)
```

Also, is the mainnet supposed to be down currently?

**Comment by azhang** [comment]
Ah I see, https://github.com/maticnetwork/eth-dagger-examples/blob/master/examples/multisig-wallet.js has updated example :)

**Comment by anurag-arjun** [comment]
We will fix this as soon as possible. Also, mainnet is working now. Please check again.

**Comment by jdkanani** [comment]
@azhang Fixed in newer version 0.0.1. I'm closing the issue for now. You can reopen it if it's not working as intended.
